### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by mayailkour** [zadam/trilium-sender#7]
Are you OK with publishing Trilium Sender on [F-Droid](https://f-droid.org/), @zadam?
F-Droid is an app store containing only open-source apps that are build from source.
If yes, could you please open an issue [here](https://gitlab.com/fdroid/rfp/issues)?
The F-Droid crew will take care of the rest. 👌

FYI @Rudloff & @Poussinou

**Comment by zadam** [comment]
Hello, I don't plan to do this myself, but if someone else wants to do that, then it would be of course appreciated :)

**Comment by Poussinou** [comment]
I opened a RFP here: https://gitlab.com/fdroid/rfp/issues/937

**Comment by IzzySoft** [comment]
It would help if releases were [tagged](https://github.com/zadam/trilium-sender/releases) – so future updates can be detected automatically, and we'd know exactly which commit to build. So which one should it be for this initial version?

One more questions: Maybe you would like to use fastlane or triple-t to maintain your app's metadata (summary, description) and screenshots. That way there could be screenshots on F-Droid, too.

**Comment by gerroon** [comment]
Is this up?

 It would be nice if @zadam makes a github release at least.

**Comment by IzzySoft** [comment]
@gerroon no, it's not up yet. We're still waiting for the tags.

@zadam any chance you'll provide them?

**Comment by BeatLink** [comment]
Is there an update on this?

**Comment by IzzySoft** [comment]
There is: RFP was closed 2 years ago for inactivity. There are still no tags here, and the last commit was more than a year ago. So it seems this project rather died.

**Comment by BeatLink** [comment]
Dang. Thats a shame. Thanks

**Comment by IzzySoft** [comment]
Yeah, we were hoping for a different outcome as well at F-Droid :cry:
